packages/renderer/src/lib/kube/details/EventsTable.svelte

```diff
@@ -0,0 +1,39 @@
+<script lang="ts">
+import humanizeDuration from 'humanize-duration';
+import moment from 'moment';
+
+import type { EventUI } from '../../events/EventUI';
+
+export let events: EventUI[];
+
+$: events.sort((ev1, ev2) => ((ev1.lastTimestamp ?? Date.now()) < (ev2.lastTimestamp ?? Date.now()) ? -1 : 1));
+function getAgeAndCount(event: EventUI): string {
+  let result = `${humanizeDuration(moment().diff(event.lastTimestamp), { round: true, largest: 1 })}`;
+  if ((event.count ?? 0) > 1) {
+    result += ` (${event.count}x over ${humanizeDuration(moment().diff(event.firstTimestamp), { round: true, largest: 1 })})`;
+  }
+  return result;
+}
+</script>
+    
+<tr>
+  <td colspan="2">
+    <table class="w-full ml-2.5">
+      <tbody>
+        <tr>
+          <th align="left">Type</th><th align="left">Reason</th><th align="left">Age</th><th align="left">From</th><th align="left">Message</th>
+        </tr>
+        {#each events as event}
+          <tr>
+            <td>{event.type}</td>
+            <td>{event.reason}</td>
+            <td>{getAgeAndCount(event)}</td>
```

Comment:
We should be using `TableDurationColumn` here, as the age will not be refresh otherwise

Reply:
The `TableDurationColumn` component is specific to the `ui-svelte`'s `Table` component, but here a simple HTML `table` is used. Also, this component is able to display a simple `Date` only, here we want to display something more elaborate (`19 seconds (2x over 25 seconds)` for example)

I agree that refreshing the date would be nice, but I would prefer to do it in a follow-up PR, as IMO it in non blocking (the events themselves are refreshed, so the user will still see the new age when the events are modified)